package.json

```diff
@@ -10,14 +10,18 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "start": "nodemon server/index.js",
-    "seed": "mysql -u root < schema.sql",
-    "shell": "mysql -u root"
+    "schema": "mysql -u root < schema.sql",
+    "shell": "mysql -u root",
+    "seed": "node fakerData.js"
```

Comment:
I think you can probably combine some scripts here to create a single script to seed your database. Check out using the && operator in a script. It will run a certain command, wait for it to finish, and then run the next one.
```suggestion
    "seed": "node fakerData.js && mysql -u root < schema.sql"
```

Reply:
That sounds way better than what I did i'll go fix that for sure!